src/executables/master_heartbeat.c

```diff
@@ -3686,8 +3686,20 @@ hb_resource_job_change_mode (HB_JOB_ARG * arg)
   rv = pthread_mutex_lock (&hb_Resource->lock);
   for (proc = hb_Resource->procs; proc; proc = proc->next)
     {
-      if (proc->type != HB_PTYPE_SERVER)
-	{
+      if (proc->type != HB_PTYPE_TRAN_SERVER)
```

Comment:
Q1. How does this change make the server writable?
Q2. Does it allow all PTS to write? (It's OK, I just want to understand. or it needs to be described?)

Reply:
A1. This job is invoked during failover. 
 When a node becomes the master node, this part finds the TS process among the processes registered in cub_master and sends a request to enable write transactions. TS receives this request and changes the value of the global variable (db_Disable_modification) that was blocking write transactions.

With the removal of the HB_PTYPE_SERVER type, this job was not executed.

A2. No, only the PTS in the master node.